packages/clerk-js/src/ui/elements/IdentityPreview.tsx

```diff
@@ -79,7 +79,7 @@ const UsernameOrEmailIdentifier = (props: any) => {
       <UserAvatar
         boxElementDescriptor={descriptors.identityPreviewAvatarBox}
         imageElementDescriptor={descriptors.identityPreviewAvatarImage}
-        imageUrl={props.avatarUrl}
+        avatarUrl={props.avatarUrl}
```

Comment:
❓ are we sure this is an internal API? Also since internally the `<Avatar/>` component uses the `imageUrl` prop, shouldn't we use the `imageUrl`  instead of `avatarUrl` ?

Reply:
This change is correct, the Avatar component sometimes needs to fallback to avatarUrl depending on where its used 

@desiprisg please confirm :)